src/ui/public/courier/courier.js

```diff
@@ -61,9 +61,9 @@ uiModules.get('kibana/courier').service('courier', ($rootScope, Private) => {
         searchPoll.pause();
 
         // And abort all pending requests.
-        _.invoke(requestQueue, 'abort');
```

Comment:
I don't think the original logic actually did anything, since `_.invoke` would try to call `requestQueue.abort`, which didn't exist.